ui/lib/ldap/addon/components/page/role/credentials.hbs

```diff
@@ -11,54 +11,58 @@
 
 <hr class="is-marginless has-background-gray-200" />
 
-{{#if (eq @credentials.type "dynamic")}}
-  <Hds::Alert @type="inline" @color="warning" class="has-top-margin-m" as |Alert|>
-    <Alert.Title>Warning</Alert.Title>
-    <Alert.Description data-test-alert-description>
-      You won’t be able to access these credentials later, so please copy them now.
-    </Alert.Description>
-  </Hds::Alert>
-{{/if}}
+{{#if @error}}
+  <Page::Error @error={{@error}} />
+{{else}}
```

Comment:
It's a bit tough to see from the diff but I just moved the existing block under the else condition and added the `Page::Error` component.